.circleci/config.yml

```diff
@@ -88,11 +83,6 @@ jobs:
       - STACK_FILE: "stack-8.8.4.yaml"
     <<: *defaults
 
-  ghc-8.10.2:
-    environment:
-      - STACK_FILE: "stack-8.10.2.yaml"
-    <<: *defaults
-
```

Comment:
Will our future plan be supporting the latest two minor versions per major one?

Reply:
I was not thinking in make it a hard rule for future, but seemed to be time to do it.
Maybe we should make explicit some policy, although it should be flexible, not all versions are equally reliable, popular, etc.

Reply:
@ailrThinking it twice we have to remove them from elsewhere and those changes maybe should too big to be included herem so i will open an issue and a pr.
That said, do you think we should revert the removing of stack.yaml's here and do it in that future pr?

Reply:
Yes, IMO `stack-<version>.yaml`s are one of the immediate hints whether we support some version or not.

Reply:
I ve reverted that change and make them work as well, opening a dedicated issue to discuss drop support for 8.8.2 and 8.10.2